padding.c

```diff
@@ -421,10 +422,11 @@ bam_hdr_t * fix_header(bam_hdr_t *old, faidx_t *fai)
 
             /* Parse the @SQ lines */
             while (cp != end) {
-                if (end-cp >= 2 && strncmp(cp, "LN", 2) == 0) {
+                if (!ln_found && end-cp >= 2 && strncmp(cp, "LN", 2) == 0) {
```

Comment:
This will ignore any extra LN tags, so they will be copied into the new file by lines 447 to 452.  It might be better to skip them and not copy them over.